### PR TITLE
Add test script for null_route_helper

### PR DESCRIPTION
**Pull request by bingwang-ms**
Signed-off-by: bingwang <bingwang@microsoft.com>

### Description of PR

Summary:
PR https://github.com/Azure/sonic-utilities/pull/1718 introduced a new helper script ```null_route_helper```. This PR is to implement test cases for the new script.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
This PR is to add test cases for ```null_route_helper```

#### How did you do it?
1. Create ACL tables on all uplink interfaces
2. Create some ACL rules as production does
3. Add block/unblock rules with ```null_route_helper``` script, and verify if packets are dropped or forwarded as expected.

#### How did you verify/test it?
Verified on SN4600, T0 testbed.

#### Any platform specific information?
No.

#### Supported testbed topology if it's a new test case?
Running on T0 topo.

### CODE REVIEW

File: tests/acl/null_route/test_null_route_helper.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        ip_ver = ipaddress.ip_network(unicode(src_ip), False).version
+        logger.info("Testing with src_ip = {} action = {} expected_result = {}"
+                .format(src_ip, action, expected_result))
+        pkt, exp_pkt = generate_packet(src_ip, DST_IP[ip_ver], router_mac)
+        if action != "":
+            rand_selected_dut.shell(NULL_ROUTE_HELPER + " " + action)
+        assert(send_and_verify_packet(ptfadapter, pkt, exp_pkt, random.choice(ptf_t1_interfaces), rx_port, expected_result))
+
+
+
```
Comment (qiluo-msft) [inline tests/acl/null_route/test_null_route_helper.py:183]: Remove extra empty lines

File: tests/acl/null_route/test_null_route_helper.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    ptfhost.shell("ifconfig eth{} 0.0.0.0".format(dst_ports['port']))
+    ptfhost.shell("ifconfig eth{} inet6 del {}".format(dst_ports['port'], dst_ports[6]))
+
+
+def generate_packet(src_ip, dst_ip, dst_mac):
+    """
+    Build ipv4 and ipv6 packets/expected_packets for testing.
+    """
+    pkt = None
+    exp_pkt = None
```
Comment (qiluo-msft) [inline tests/acl/null_route/test_null_route_helper.py:121]: You can remove the init lines?

Comment (bingwang-ms) [reply tests/acl/null_route/test_null_route_helper.py]: Updated. Thanks

File: tests/acl/null_route/test_null_route_helper.py (hunk 1 above)
Comment (bingwang-ms) [reply tests/acl/null_route/test_null_route_helper.py:180]: Updated. Thanks

**Review by qiluo-msft — APPROVED** [review]

**Comment by bingwang-ms** [comment]
Please hold on merging this PR until ```null_route_helper``` script is checked in.

**Pull request merged**